### PR TITLE
fix release please configs

### PR DESCRIPTION
**Pull request by shemogumbe**

### CODE REVIEW

File: release-please-config.json [hunk 1]
```diff
@@ -6,9 +6,9 @@
       ".github",
       ".vscode"
     ],
-    "release-type": "python",
+    "release-type": "simple",
     "include-component-in-tag": false,
-    "include-v-in-tag": true,
+    "include-v-in-tag": false,
```
Comment (baywet) [inline release-please-config.json]: We've been adding a v in the tag since the beginning. Why the change?

**Review by baywet — CHANGES_REQUESTED** [review]

File: release-please-config.json (hunk 1 above)
Comment (shemogumbe) [reply release-please-config.json]: This was an unneccesary change, intended to change the release-type and the hash to the last release commit

**Review by andrueastman — APPROVED** [review]

**Review by samwelkanda — APPROVED** [review]

**Review by baywet — APPROVED** [review]
Thank you for making the changes!

**Comment by baywet** [comment]
I didn't see it originally but it seems the space between the closing parenthesis and the colon makes the commit "invalid" from a conventional commit/release please perspective, not triggering the release.
